sycl/doc/extensions/proposed/sycl_ext_oneapi_p2p.asciidoc

```diff
@@ -0,0 +1,157 @@
+= sycl_ext_oneapi_myextension
```

Comment:
Update this title.

BTW, maybe it's better to write out the name as "sycl_ext_oneapi_peer_access"?  That might be more clear when someone is reading through a list of the extensions.  If you decide to do this, also change the name of the file and the name of the feature-test macro.